pkg/csi_driver/node.go

```diff
@@ -396,3 +384,27 @@ func gcsFuseSidecarContainerImage(pod *corev1.Pod) string {
 	}
 	return ""
 }
+
+func addFuseMountOptions(identityProvider string, identityPool string, fuseMountOptions []string, vc map[string]string, hostNetworkEnabled bool, enableSidecarBucketAccessCheckForSidecarVersion bool, enableCloudProfilerForVersion bool) ([]string, error) {
```

Comment:
Please add function comment . 
Is there a reason why we didn't put everything into addFuseMountOptions. For example, testing the MachineTypeAutoConfig?